pkg/network/tracer/connection/failure/matching.go

```diff
@@ -32,7 +32,7 @@ var (
 	}
 
 	telemetryModuleName = "network_tracer__tcp_failure"
-	mapTTL              = 60 * time.Second.Nanoseconds()
+	mapTTL              = 10 * time.Millisecond.Nanoseconds()
```

Comment:
This is drastic increase from before. Is such a big change justified?

Reply:
discussed offline and we are going to leave this and set the frequency of map cleaner runs to 1s